src/getFiltred.ts

```diff
@@ -0,0 +1,45 @@
+import { Todo } from './types/Todo';
+
+export enum SelectStatus {
+  Completed = 'completed',
+  All = 'all',
+  Active = 'active',
+}
+
+type Filtred = {
+  query: string;
+  todoStatus: SelectStatus;
+};
+
+export const getFiltred = (
+  todos: Todo[],
+  options: Filtred,
+) => {
+  const { todoStatus, query } = options;
+  let filtredTodos = [...todos];
+
+  if (todoStatus !== SelectStatus.All) {
+    filtredTodos = filtredTodos.filter(todo => {
+      switch (todoStatus) {
+        case SelectStatus.Completed:
+          return todo.completed;
+
+        case SelectStatus.Active:
+          return !todo.completed;
+
+        default:
+          return true;
+      }
+    });
+  }
+
+  if (query) {
+    filtredTodos = filtredTodos.filter(todo => {
+      return todo.title
+        .toLowerCase()
+        .includes(query.toLowerCase());
```

Comment:
Also, I frogot note this in previous review, you should trim search query value, because now if I enter only spaces, I get it:
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/54936044/a4a06086-7808-42f1-b5ab-9b4f82d7b63f)